diff -ur fontconfig-2.11.1.orig/configure.ac fontconfig-2.11.1/configure.ac
--- fontconfig-2.11.1.orig/configure.ac	2014-03-24 15:03:12.000000000 +0900
+++ fontconfig-2.11.1/configure.ac	2014-04-05 14:45:30.701711217 +0900
@@ -150,7 +150,7 @@
 # Checks for library functions.
 AC_FUNC_VPRINTF
 AC_FUNC_MMAP
-AC_CHECK_FUNCS([link mkstemp mkostemp _mktemp_s mkdtemp getopt getopt_long getprogname getexecname rand random lrand48 random_r rand_r readlink regcomp regerror regexec regfree fstatvfs fstatfs lstat])
+AC_CHECK_FUNCS([link mkstemp mkostemp mkdtemp getopt getopt_long getprogname getexecname rand random lrand48 random_r rand_r readlink regcomp regerror regexec regfree fstatvfs fstatfs lstat])
 
 dnl AC_CHECK_FUNCS doesn't check for header files.
 dnl posix_fadvise() may be not available in older libc.
